frontend/app/routes/_public+/apply+/$id+/demographics-part1.tsx

```diff
@@ -26,58 +32,165 @@ export const meta: MetaFunction<typeof loader> = mergeMeta((args) => {
+    return t(`demographics-oral-health-questions:part1.error-message.${errorI18nKey}` as any);
+  }
+
+  const errorMessages = {
+    bornType: getErrorMessage(actionData?.errors.bornType?._errors[0]),
+    firstNations: getErrorMessage(actionData?.errors.firstNations?._errors[0]),
+    disability: getErrorMessage(actionData?.errors.disabilityType?._errors[0]),
+  };
+
+  const errorSummaryItems = createErrorSummaryItems(errorMessages);
+  useEffect(() => {
+    if (actionData?.formData && hasErrors(actionData.formData)) {
+      scrollAndFocusToErrorSummary(errorSummaryId);
+    }
+  }, [actionData]);
 
   return (
     <>
       <p className="mb-6">{t('apply:demographics-oral-health-questions.part1.paragraph1')}</p>
       <p className="mb-6">{t('apply:demographics-oral-health-questions.part1.paragraph2')}</p>
-      <Form method="post">
+      {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+      <Form method="post" noValidate className="space-y-6">
         {bornTypes.length > 0 && (
           <div className="my-6">
             <InputRadios
-              id="born-type"
+              id="bornType"
               name="bornType"
               legend={t('apply:demographics-oral-health-questions.part1.question1')}
               options={bornTypes.map((bornType) => ({
+                defaultChecked: state?.bornType === bornType.id,
                 children: getNameByLanguage(i18n.language, bornType),
                 value: bornType.id,
               }))}
-              required
             />
           </div>
         )}
+
+        {firstNationsTypes.length > 0 && <InputRadios id="firstNations" legend={t('apply:demographics-oral-health-questions.part1.first-nations')} name="firstNations" options={options} required></InputRadios>}
+
         {disabilityTypes.length > 0 && (
           <div className="my-6">
             <InputRadios
-              id="disability-type"
+              id="disabilityType"
               name="disabilityType"
               helpMessagePrimary={t('apply:demographics-oral-health-questions.part1.question4-note')}
               legend={t('apply:demographics-oral-health-questions.part1.question4')}
               options={disabilityTypes.map((disabilityType) => ({
+                defaultChecked: state?.disabilityType === disabilityType.id,
                 children: getNameByLanguage(i18n.language, disabilityType),
                 value: disabilityType.id,
               }))}
-              required
             />
           </div>
         )}
         <div className="flex flex-wrap items-center gap-3">
-          <ButtonLink id="back-button" to="/personal-information">
-            <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
+          <ButtonLink id="cancel-button" to={`/apply/${id}/demographics`}>
+            <FontAwesomeIcon icon={faChevronLeft} className="pl-2" />
```

Comment:
No need to change the previous CSS and `id`:

```suggestion
          <ButtonLink id="back-button" to={`/apply/${id}/demographics`}>
            <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
```